python-frontend/src/test/java/org/sonar/python/semantic/v2/typeshed/ModuleSymbolToDescriptorConverterTest.java

```diff
@@ -34,7 +34,7 @@ class ModuleSymbolToDescriptorConverterTest {
 
   @Test
   void test() {
-    var converter = new ModuleSymbolToDescriptorConverter();
+    var converter = new ModuleSymbolToDescriptorConverter(ProjectPythonVersion.currentVersions());
```

Comment:
Couldn't that be source of nondeterminism if `ProjectPythonVersion#setCurrentVersions` is called in another test? Isn't it better to call `PythonVersionUtils#allVersions` explicitly?